Automattic/wp-calypso
# composite-checkout: Processor and validation for multi-partner card processor

#### Changes proposed in this Pull Request

Depends on #43841.

* Adds a processor function for the multi-partner card payment method.

#### Testing instructions

This makes significant changes to the stripe processor and adds ebanx for Brazilian card payments, so we need to check all the new card related flows. To test ebanx, apply code-D45663 to your sandbox and at the payment method step, uncheck the box to change your contact info and select Brazil. Stripe has test cards we can use at https://stripe.com/docs/testing as does ebanx at https://developer.ebanx.com/docs/resources/testCreditCards/.

- Stripe, non-3DS, payment success
- Stripe, non-3DS, payment failure
- Stripe, 3DS, payment success
- Stripe, 3DS, payment failure
- Ebanx, payment success
- Ebanx, payment failure

We also need to jerk test the validation of the Ebanx contact fields.
